extensions/azure-keyvault/deployment/src/test/java/io/quarkiverse/azure/keyvault/secret/deployment/KeyVaultSecretAsyncClientCDITest.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkiverse.azure.keyvault.secret.deployment;
+
+import jakarta.inject.Inject;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.azure.security.keyvault.secrets.SecretClient;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class KeyVaultSecretAsyncClientCDITest {
+
+    @Inject
+    SecretClient client;
```

Comment:
This should be `SecretAsyncClient`?

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/1b0f9b4ce9d8e8cfba02237c818ba1bdb4c90574.